### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by rasika-chivate**
Update the readme for Everest.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,61 +1,137 @@
...
+    rm everestctl-darwin-arm64
+    ```
+
+    **macOS (Intel CPU)**
+
+    ```sh
+    curl -sSL -o everestctl-darwin-amd64 https://github.com/percona/percona-everest-cli/releases/latest/download/everestctl-darwin-amd64
+    sudo install -m 555 everestctl-darwin-amd64 /usr/local/bin/everestctl
+    rm everestctl-darwin-amd64
+    ```
```
Comment (recharte) [inline README.md:52]: I would move this entire part to point 1. in the Install section

**Review by oksana-grishchenko — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,61 +1,136 @@
...
+    ```sh 
+    kubectl get nodes
+    ```
+
+## Install Percona Everest
+
+To install and provision Percona Everest to Kubernetes:
+
+
+1. Download the latest version of Everest CLI by visiting the latest release page in [this repository](https://github.com/percona/percona-everest-cli/releases/latest).
```
Comment (recharte) [inline README.md:47]:
```suggestion
1. Install the latest version of the Everest CLI by running the following commands:
```

**Review by recharte — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1,61 +1,143 @@
...
-## Creating Kubernetes cluster
+      ```sh
+      everestctl install --namespaces <namespace-name1>,<namespace-name2> --operator.mongodb=true --operator.postgresql=true --operator.xtradb-cluster=true --skip-wizard
+      ```
+    
+    Replace `<namespace-name>` with the desired name for your namespace.
 
-You must have a publicly accessible Kubernetes cluster to use Everest. EKS or GKE is recommended, as it may be difficult to make it work with local installations of Kubernetes such as minikube, kind, k3d, or similar products. Everest does not help with spinning up a Kubernetes cluster but assists with installing all the necessary components for Everest to run.
+    !!! note
+        Ensure that you copy the authorization token displayed on the terminal in this step. You will need this token to log in to the Percona Everest UI.
```
Comment (recharte) [inline README.md:98]: @rasika-chivate I don't think this is rendered correctly in GitHub, AFAIK this ain't standard markdown.

File: README.md [hunk 4]
```diff
@@ -1,61 +1,143 @@
...
+    ```sh
+    everestctl install
+    ```
+
+    Enter the specific names for the namespaces you want Everest to manage, separating each name with a comma.
+
+    !!! note 
+        Make sure that you enter at least one namespace.
+
+    Alternatively, you can set multiple namepaces in the headless mode:
```
Comment (recharte) [inline README.md:89]: Same here

**Review by recharte — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -1,61 +1,143 @@
-## Welcome to Percona Everest
+# Percona Everest
 
-Percona Everest is an open source Database-as-a-Service solution that automates day-one and day-two operations for Postgres, MySQL, and MongoDB databases within Kubernetes clusters.
+!!! warning "Important"
+    Percona Everest is now in **Beta**, and we need your [feedback](https://forums.percona.com/c/percona-everest/81)! We want you to be a part of this journey and help us shape our product.
```
Comment (recharte) [inline README.md:4]: Same here

File: README.md (hunk 3 above)
Comment (rasika-chivate) [reply README.md:98]: The formatting is correct. Let me check why its not rendering properly.

File: README.md (hunk 4 above)
Comment (rasika-chivate) [reply README.md:89]: The formatting is correct. Let me check why its not rendering properly.

**Review by recharte — APPROVED** [review]

**Review by recharte — APPROVED** [review]

**Pull request merged**